html/janus.js

```diff
@@ -1108,7 +1108,13 @@ function Janus(gatewayCallbacks) {
 			ws.addEventListener('message', onUnbindMessage);
 			ws.addEventListener('error', onUnbindError);
 
-			ws.send(JSON.stringify(request));
+                        if (ws.readyState === 1) {
+                            ws.send(JSON.stringify(request));
+                        }
+                        else {
+                            onUnbindError();
+                        }
+
```

Comment:
Please use tabs, not spaces, for indentation. The `else` should be on the same line as the closing bracket too.

Reply:
Both fixed now.

Reply:
They don't seem fixed to me... I still see spaces used for indentation.

Reply:
Sorry, it was using 4-char indentation and I've changed it to 8 (so all tabs).

Reply:
It must not be char-indentation, only tab-indentation. I still see whitespaces.

Reply:
Edit: apologies, github was showing me an old thing... seems ok now! :+1: